app.rb

```diff
@@ -0,0 +1,172 @@
+require './book'
+require './student'
+require './teacher'
+require './rental'
+require './person'
+
+class App
+  attr_reader :people, :books
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def show_menu
+    puts 'Welcome to school library app!'
+    puts ''
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person ID'
+    puts '7 - Exit'
+    @number = gets
+    until @number.to_i.between?(1, 7)
+      puts 'Please write a number between 1 and 7'
+      @number = gets
+    end
+  end
+
+  def options
+    case @number.to_i
+    when 1
+      list_all_books
+      run
+    when 2
+      list_all_people
+      run
+    when 3
+      create_person
+      run
+    when 4
+      create_book
+      run
+    when 5
+      create_rental
+      run
+    when 6
+      list_all_rentals_for_id
+      run
+    end
+  end
+
+  def run
+    show_menu
+    if @number.to_i == 7
+      exit
+    else
+      options
+    end
+  end
```

Comment:
- [ ] Please 🙏🏽 move this code to the main.rb file for OOP it is better to have methods assigned according to the class in this case class app will create the functions for each option and the class main will show the menu of options to select.